scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -49,41 +50,79 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
     ).map(_.find(_.path.toString.contains("mill-jsbridge")).get)
   }
 
-
   def scalaJSLinkerClasspath: T[Loose.Agg[PathRef]] = T{
+    val commonDeps = Seq(
+      ivy"org.scala-js::scalajs-tools:${scalaJSVersion()}",
+      ivy"org.scala-js::scalajs-sbt-test-adapter:${scalaJSVersion()}",
+      ivy"org.scala-js::scalajs-test-interface:${scalaJSVersion()}"
```

Comment:
@rockjam chatted with @sjrd, this line is the culprit. `scalajs-test-interface` should live in `scalaJSTestDeps` (which you already have it) and `scalaJSLinkerClasspath` should instead have the SBT `org.scala-sbt:test:interface:1.0`, which we should not need to add since `scalajs-sbt-test-adapter` should pull it in for us.

org.scala-sbt | test-interface | 1.0
-- | -- | --




org.scala-sbt | test-interface | 1.0
-- | -- | --

Reply:
I see. That was confusing. Thanks for help